CHANGES.rst

```diff
@@ -11,7 +11,9 @@ Unreleased
 -   Remove code that was marked deprecated.
 -   Use :pep:`451` API to load templates with
     :class:`~loaders.PackageLoader`. :issue:`1168`
-
+-   Add required attribute to blocks that must be
```

Comment:
Make "required" a code item by surrounding it with double backticks.

This can be wrapped at 72 characters, all the other lines just happened to be short.